src/applications/claims-status/components/claim-files-tab/DocumentsFiled.jsx

```diff
@@ -174,7 +174,11 @@ function DocumentsFiled({ claim }) {
                   {item.text && (
                     <div className="vads-u-margin-top--0 vads-u-margin-bottom--1">
                       {reviewed(item.text) && (
-                        <i className="fa fa-check-circle docs-filed-icon" />
+                        <va-icon
+                          size={4}
+                          icon="see Storybook for icon names: https://design.va.gov/storybook/?path=/docs/uswds-va-icon--default"
```

Comment:
Should this text be replaced with one of the icon names available at the URL here?

While we are making changes to remove sighted language, would it be prudent to add the `srtext` property to this icon as well?

Reply:
Is there an auto thing that is adding this change? I didnt add this icon change but I've noticed its been added to multiple of my prs and then I have to undo it. If you guys want us to update from using <i> to <va-icon> throughout our project then we should probably have a specific ticket for this work. 

Going to roll this change back since I did not intend for this to be added.

Reply:
Looks like this was an issue https://dsva.slack.com/archives/CBU0KDSB1/p1714073709459219
Additionally, looks like va-icon is still being worked on reading convos today so we shouldnt be using it yet.